### ISSUE TITLE
Update `fetch_translations.js` to use new Primero adaptor functions for requests to `/lookups` and `/forms` endpoints

### ISSUE THREAD

**Issue by aleksa-krolls** [OpenFn/unicef-thailand#61]
## Background, context, and business value
Update job [fetch_translations.js](https://github.com/OpenFn/primero-thailand/blob/staging/jobs/child-shield/fetch_translations.js) to use the new Primero adaptor functions for accessing the `lookups` and `forms` API. 

This job is currently running using the HTTP adaptor. After adding the new functions to the Primero adaptor, we want to update this job so that it is using this Primero adaptor. 

## The specific request, in as few words as possible
See issue #30 for details on state, configuration, and how to test... or let @daissatou2 know if you need any help testing. 

## expression
Please make all changes on *staging* branch: https://github.com/OpenFn/primero-thailand/blob/staging/jobs/child-shield/fetch_translations.js

## To test
Make sure this is running OK locally first. 

Then update the adaptor for the job on platform to make sure it's running correctly there as well: https://www.openfn.org/projects/primero-thailand-io-unicef---staging/jobs/zXxiCH

Let daissatou2 know if you need any help testing.

**Comment by aleksa-krolls** [comment]
@mtuchi is this adaptor change ready to go and deployed on platform as a new version? 

I saw you submitted [this PR](https://app.zenhub.com/workspaces/implementation-engineering-600030af88f2bb00155d46a5/issues/openfn/primero-thailand/62) for the `fetch_translations.js` job... but today we implemented the `noop` changes on the job in the staging branch. 

Can you therefore please update this PR to include the latest logic? See my comment: https://app.zenhub.com/workspaces/implementation-engineering-600030af88f2bb00155d46a5/issues/openfn/primero-thailand/62
And then let me know when the PR is ready to merge and what adaptor version we should set the job as on platform?

**Comment by mtuchi** [comment]
> @mtuchi is this adaptor change ready to go and deployed on platform as a new version?
> 
> I saw you submitted [this PR](https://app.zenhub.com/workspaces/implementation-engineering-600030af88f2bb00155d46a5/issues/openfn/primero-thailand/62) for the `fetch_translations.js` job... but today we implemented the `noop` changes on the job in the staging branch.
> 
> Can you therefore please update this PR to include the latest logic? See my comment: https://app.zenhub.com/workspaces/implementation-engineering-600030af88f2bb00155d46a5/issues/openfn/primero-thailand/62 And then let me know when the PR is ready to merge and what adaptor version we should set the job as on platform?

@aleksa-krolls working on a fix, Will push the PR before end of day

**Comment by mtuchi** [comment]
> > @mtuchi is this adaptor change ready to go and deployed on platform as a new version?
> > I saw you submitted [this PR](https://app.zenhub.com/workspaces/implementation-engineering-600030af88f2bb00155d46a5/issues/openfn/primero-thailand/62) for the `fetch_translations.js` job... but today we implemented the `noop` changes on the job in the staging branch.
> > Can you therefore please update this PR to include the latest logic? See my comment: https://app.zenhub.com/workspaces/implementation-engineering-600030af88f2bb00155d46a5/issues/openfn/primero-thailand/62 And then let me know when the PR is ready to merge and what adaptor version we should set the job as on platform?
> 
> @aleksa-krolls working on a fix, Will push the PR before end of day

@aleksa-krolls this PR #62 is ready for your review

**Comment by aleksa-krolls** [comment]
@mtuchi confirming I can test this now with `v2.8.0`? Or is more work needed (and should we move this to backlog, if yes)? 
Here is our [latest Slack thread](https://openfn.slack.com/archives/C03EQQB5W7R/p1666017687944539) about this.

**Comment by mtuchi** [comment]
> @mtuchi confirming I can test this now with `v2.8.0`? Or is more work needed (and should we move this to backlog, if yes)? Here is our [latest Slack thread](https://openfn.slack.com/archives/C03EQQB5W7R/p1666017687944539) about this.

@aleksa-krolls yes it's ready for testing

**Comment by aleksa-krolls** [comment]
@daissatou2 after you test, please share your feedback (if any) on here. Thanks!

**Comment by daissatou2** [comment]
@mtuchi please take a look at this [error](https://www.openfn.org/projects/primero-thailand-io-unicef---staging/runs/063502b6-7744-710d-a2ae-814b1fa55042) on platform: 

`TypeError [Error]: Cannot read properties of undefined (reading 'reduce')`

You can try to reproduce locally using the cursor currently in job 1: `2022-09-28T00:43:07.000Z`. 

I've reverted job 2 on the platform to yesterday's commit (using the HTTP adaptor) in case the client is testing tomorrow morning. Please submit a PR once this is ready for testing again.

**Comment by mtuchi** [comment]
> @mtuchi please take a look at this [error](https://www.openfn.org/projects/primero-thailand-io-unicef---staging/runs/063502b6-7744-710d-a2ae-814b1fa55042) on platform:
> 
> `TypeError [Error]: Cannot read properties of undefined (reading 'reduce')`
> 
> You can try to reproduce locally using the cursor currently in job 1: `2022-09-28T00:43:07.000Z`.
> 
> I've reverted job 2 on the platform to yesterday's commit (using the HTTP adaptor) in case the client is testing tomorrow morning. Please submit a PR once this is ready for testing again.
@daissatou2 
Fixed on #73 , If you merge make sure to use `language-primero v2.9.1`, See my environment 👇🏽 
```
╭─────────────────────────────────────────────╮
│ ◲ ◱  @openfn/core#v1.4.8 (Node.js v18.10.0) │
│ ◳ ◰          @openfn/language-primero@2.9.1 │
╰─────────────────────────────────────────────╯
```
cc @aleksa-krolls
